TurkishFilter/sections/specific.txt

```diff
@@ -63,6 +63,7 @@ sporx.com##.hk_bar
 ||iddaa.sporx.com/web/kutu_iddaa_bulten.php
 ||iddaa.sporx.com/web/kutu_kral_kupon.php
 ||iddaa.sporx.com/web/kutu_ozel_etkinlik.php
+||blogger.googleusercontent.com/img/a/
```

Comment:
This rule is very wide `||blogger.googleusercontent.com/img/a/$domain=randomovie.co` would be better but anyway I've already fixed the issue.